### PR TITLE
Added Operator Section to GREL Page

### LINKED ISSUE #289: Documentation of GREL operators is lacking
Although the functionality of operators in GREL is quite intuitive, there is some overloading and special functionality that is not documented in an easy way to find. For example, the overloading of the + operator for string concatenation is only mentioned in the 'Basics' section, while the != (not equal to) operator is not mentioned at all. The special behaviour of the / operator for ints is nicely mentioned at the top of the Math functions section, but the % (integer modulus) operator gets no mention (though it does work - I use it a lot!). It would be great if there were either a separate section on operators, or else that all operators are described in each object type section i.e. at the top of each of 'String functions', and 'Math Functions' (there are no overloaded operators for array types, sadly, so no need for a mention in that section). I'm not aware of any operators that work for Date objects (presumably == and != work, but it might be logical to assume the relational operators like < and > work for these and someone might look to the documentation to find out.

### PR DESCRIPTION
**Pull request by aayushk9**
Fixes #289 
Introduced an 'Operator' section to the GREL page, improving user accessibility to information about operators in GREL.

### CODE REVIEW

**Comment by aayushk9** [comment]
@wetneb Can you review you this PR?

**Comment by aayushk9** [comment]
@wetneb Can you review this PR?

File: docs/manual/grelfunctions.md [hunk 1]
```diff
@@ -35,6 +35,30 @@ Uses the logical operator NOT on a boolean to output a boolean. For example, `no
...
+
+When using the `%` operator, if both operands are numbers such as `1 % 2` the result will be a whole number. However, if either or both of the operands are floating-point numbers like `1.0 % 2` they will be promoted to floating point and the result will also be in floating-point format. It's important to note that the `%` operator may not behave as expected with floating-point numbers due to precision issues.
+
+###### Multiplication {#multiplication}
+
+The behavior of the `*` operator is nuanced based on the data types of the operands. When both operands are integers such as `1 * 2`, the result is an integer. Conversely, if either or both operands are floating-point numbers the result becomes a floating-point number. You can use simple evaluations such as `3.5 * 2`
+
+#### Relational Operators {#relational-operators}
+
+`==` and `!=` operators can be used for string equality and inequality. For example, `"a" == "b"` returns false and `"a" != "b"` returns true.
```
Comment (tfmorris) [inline docs/manual/grelfunctions.md:52]: These work for any data type, not just strings.

File: docs/manual/grelfunctions.md [hunk 2]
```diff
@@ -35,6 +35,30 @@ Uses the logical operator NOT on a boolean to output a boolean. For example, `no
...
+
+The `<` operator checks if the left operand is less than the right operand and `<=` checks if it's less than or equal to. Similarly `>` verifies if the left operand is greater than the right and `>=` checks if it's greater than or equal to. These comparison operators yield Boolean values (true or false)
```
Comment (tfmorris) [inline docs/manual/grelfunctions.md:54]: These operators work for number, strings, and dates.

File: docs/manual/grelfunctions.md [hunk 3]
```diff
@@ -35,6 +35,30 @@ Uses the logical operator NOT on a boolean to output a boolean. For example, `no
...
+
+#### References {#references}
+
+- [String Concatenation](https://openrefine.org/docs/manual/grel#basic)
+- [Division('/') Documentation](https://openrefine.org/docs/manual/grelfunctions#math-functions)
+- [Logical Operators](https://openrefine.org/docs/manual/grelfunctions#boolean-functions)
```
Comment (tfmorris) [inline docs/manual/grelfunctions.md:60]:
```suggestion
- [Logical Functions](https://openrefine.org/docs/manual/grelfunctions#boolean-functions)
```

File: docs/manual/grelfunctions.md [hunk 4]
```diff
@@ -35,6 +35,30 @@ Uses the logical operator NOT on a boolean to output a boolean. For example, `no
 
 Uses the logical operator XOR (exclusive-or) on two or more booleans to output a boolean. Evaluates multiple statements, then returns true if only one of them is true. For example, `(1 < 3).xor(1 < 7)` returns false because more than one of the conditions is true.
 
+### Operators {#operators}
+
+#### Arithmetic Operators {#arithmetic-operators}
+
```
Comment (tfmorris) [inline docs/manual/grelfunctions.md:41]: Move (or copy?) the division documentation from the Math Functions page to here.

**Review by tfmorris — CHANGES_REQUESTED** [review]
Thanks a lot for the contribution! Operators are distinct from Functions, so this would probably fit better on the GREL page than on the GREL Functions page.

I made a few other small comments inline.

**Comment by aayushk9** [comment]
Hello @tfmorris
Thank you for pointing that out. I'll make the required changes

**Comment by ostephens** [comment]
LGTM thank you @aayushk9 for the contribution and @tfmorris for review

**Comment by wetneb** [comment]
@aayushk9 it looks like you deleted your fork before this pull request was formally accepted, which means that your changes have not yet made it into OpenRefine's website. Was that intended?
It could still be possible to salvage your work by recovering it from [this view](https://github.com/OpenRefine/openrefine.org/pull/295/files), but it's a little more involved!

**Comment by aayushk9** [comment]
@wetneb Apologies for the inconvenience. I mistakenly deleted the forked repository. To rectify this, I've made all the necessary changes and created a new pull request. You can find the new pull request [here](https://github.com/OpenRefine/openrefine.org/pull/296)

**Pull request closed without merging**